packages/shared/lib/services/sync/sync.service.ts

```diff
@@ -319,9 +320,23 @@ export const getSyncs = async (nangoConnection: Connection): Promise<Sync[]> =>
             `${SYNC_SCHEDULE_TABLE}.schedule_id`
         );
 
-    const syncsWithSchedule = result.map((sync) => {
+    const syncsWithSchedule = result.map(async (sync) => {
         const { schedule_id } = sync;
         const schedule = scheduleResponse?.schedules.find((schedule) => schedule.scheduleId === schedule_id);
+        // if a disagreement trust temporal
+        if (schedule?.info?.paused && sync.schedule_status !== SyncStatus.PAUSED) {
```

Comment:
can we have other disagreement where status would be different from PAUSED

Reply:
No, don't think so. A schedule in temporal is either PAUSED or RUNNING